CONTRIBUTING.md

```diff
@@ -228,32 +229,40 @@ only pull requests targeted directly against master.
 - Tag the release (use `git tag -a`) and create a release in Github
 - Delete the `RC` branches
 
-### Point Release Procedure
+### Stable Release Update Procedure
 
 At the moment, only a single major release will be supported, so all point releases will be based
 off of that release.
 
 In order to alleviate the burden for a single person to have to cherry-pick and handle merge conflicts
-of all desired backporting PRs to a point release, we instead maintain a living backport branch, where
+of all desired backporting PRs to a point release, we instead maintain a living SRU branch, where
```

Comment:
can we keep backport instead of SRU? it took me a sec to figure out what it meant.

Reply:
Please see my comment below